templates/activity.mustache

```diff
@@ -64,9 +64,40 @@
     <!-- Only display mastery for practice activities. -->
     {{#sectionid}}
     <span class="emphasis-bold answer">Chapter Mastery (<span id="chapter-mastery-title"></span>):</span><br>
-    <progress class="progress" id="chapter-mastery" value="0" max="100" data-text="0%"></progress><br>
+        <svg style="display:none;">
+            <defs>
+                <symbol id="fivestars">
+                    <path d="M12 .587l3.668 7.568 8.332 1.151-6.064 5.828 1.48 8.279-7.416-3.967-7.417 3.967 1.481-8.279-6.064-5.828 8.332-1.151z M0 0 h24 v24 h-24 v-24" fill="white" fill-rule="evenodd"/>
```

Comment:
I haven't done it in Mustache but I think it does support macros, if it does can you move these definitions to a macro, please?